internal/connection/orchestrator.go

```diff
@@ -83,26 +100,34 @@ func (o *Orchestrator) Connect() {
 	}
 }
 
-func downloadBlob(o *Orchestrator, connectionString string) string {
-	blob, err := o.executor.ExecuteSilent("curl -s --retry 3 " + connectionString)
-
+func downloadBlob(connectionString string) (string, error) {
+	res, err := req.Get(connectionString)
```

Comment:
do we still do 3 retries here?

Reply:
currently there is no retry anymore, if it should be re-added i need to implement it manually